presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -629,4 +629,20 @@ default void revokeTablePrivileges(ConnectorSession session, SchemaTableName tab
     {
         return emptyList();
     }
+
+    /**
+     * Commits partition for table creation.
```

Comment:
Maybe also explain that for connector supports partition commit, engine can do recoverable grouped execution?

Reply:
"for connector supports partition commit, engine can do recoverable grouped execution" is not exactly correct because we have other limit as well (I wish we could just claim that :D). How about "To enable recoverable grouped execution, it is required that output connector supports partition commit"?